deps/spidershim/src/v8trycatch.cc

```diff
@@ -41,7 +41,7 @@ struct TryCatch::Impl {
     // TODO: Propagate exceptions to the caller
     //       https://github.com/mozilla/spidernode/issues/51
   }
-  Isolate* Isolate() const { return isolate_; }
+  Isolate* Iso() const { return isolate_; }
```

Comment:
OK, I'll fix this later myself.  What version of gcc did you get this error with?